clientv2/client.go

```diff
@@ -178,26 +172,31 @@ func (c *Client) Post(ctx context.Context, operationName, query string, respData
 			return fmt.Errorf("failed to prepare form body: %w", err)
 		}
 
-		headers = append(headers, header{key: "Content-Type", value: contentType})
+		multipartFilesGroupsContentType = contentType
```

Comment:
@coderabbitai

Reply:
I think the mpartFilesGroupsContentType variable is redundant. Can we build the request without defining this variable?

Reply:
That would require that the req be created before the body is created, which seems more verbose.